cardano-node/README.md

```diff
@@ -116,25 +114,36 @@ Available options:
 
 ## Genesis operations
 
-The canned `scripts/genesis.sh` example provides a nice set of defaults and
-illustrates available options.  Running it will produce a `./genesis.${systemStart}` directory,
-where `${systemStart}` will be chosen 15 minutes in the future.
+### Generation
+
+The genesis generation operations will create a directory that contains:
 
-The directory will have the following content:
+  - the genesis JSON file itself (`genesis.json`)
```

Comment:
Can we document each of the items it creates? e.g Which keys are generated and what do they do etc. Doesn't have to be long. You could link to other documentation that explains this if possible. I'm not sure if this is up to date: https://cardanodocs.com/

Reply:
Done.